projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,36 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.adf-info-drawer-tab').nth(0);
+  public commentsTab = this.getChild('.adf-info-drawer-tab').nth(1);
+  public commentInputField = this.getChild('mat-form-field');
+  public commentsHeader = this.getChild('#comment-header');
+  public addCommentButton = this.getChild('[data-automation-id="comments-input-add"]');
+  public commentsList = this.getChild('.adf-comment-list-item');
+  public commentUsername = this.getChild('.adf-comment-user-name');
+  public commentTextContent = this.getChild('.adf-comment-message');
+  public commentTimestamp = this.getChild('.adf-comment-message-time');
+  public commentProfileIcon = this.getChild('.adf-comment-user-icon');
+
+
+  async checkCommentsHeaderCount() {
```

Comment:
we also set return types for functions so could you please update your functions with returned type?

Reply:
Will be fixed in the next push